src/NodaTime/Text/ZonedDateTimePatternParser.cs

```diff
@@ -82,6 +82,18 @@ public IPattern<ZonedDateTime> ParsePattern(string patternText, NodaFormatInfo f
                     'F' => ZonedDateTimePattern.Patterns.ExtendedFormatOnlyPatternImpl
                             .WithZoneProvider(zoneProvider)
                             .WithResolver(resolver),
+                    'e' => (zoneProvider == null) ? 
```

Comment:
Hmm. I need to think about this a bit more. I think it's probably most consistent to be able to pass in null as the zone provider and get a format-only pattern... but still keeping the public properties defaulting to TZDB.

I suspect we may end up needing more internal properties for this, I'm afraid. I'll try to experiment with it later in the week.

Reply:
absolutely.  no urgency at all.  if i get some time, i'll work on the serialization side of things as we discussed in the other PR.  no urgency there either.  i just figured as I have time--and before thinking about timezones drops out of my cache :)-- I'll try to push the ball along so that hopefully things are a little easier for you when you get the chance to focus on it.  but yeah, on the current rfc timeline, it looks like this is a 2022 merge regardless.  good luck with the avalanche!